components/server/src/workspace/workspace-starter.ts

```diff
@@ -1317,7 +1316,11 @@ export class WorkspaceStarter {
         const disp = new DisposableCollection();
 
         if (mustHaveBackup) {
-            result.setBackup(new FromBackupInitializer());
+            const backup = new FromBackupInitializer();
+            if (CommitContext.is(context)) {
+                backup.setCheckoutLocation(context.checkoutLocation || "");
```

Comment:
Shouldn't we default to "."? : 🤔

Reply:
Being picky here bc we had an incident after the last change in the multirepo PR

Reply:
> Shouldn't we default to "."?

Also in all cases, even if not a `CommitContext`?

Reply:
@csweichel Could you double-check here?